src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,65 @@
-export const Tabs = () => {};
+import React from 'react';
+import './Tabs.scss';
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  selectedTab,
+}) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={cn({ 'is-active': selectedTabId === tab.id })}
+          >
+            <a
+              href={`#${selectedTabId}`}
+              data-cy="TabLink"
+              onClick={() => selectedTab(tab.title)}
+            >
+              {selectedTab && tab.title}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+
+    <div
+      className="block"
+      data-cy="TabContent"
+    >
+      <>
+        <p className="content">
+          {selectedTabId && tabs.find(tab => tab.id === selectedTabId)?.content}
```

Comment:
```suggestion
          {selectedTabId && tabs.find(tab => tab.id === selectedTabId)?.content}
```

Create a separate variable for this part
`tabs.find(tab => tab.id === selectedTabId`